sql/src/main/java/org/apache/druid/sql/avatica/DruidStatement.java

```diff
@@ -350,6 +357,32 @@ public void close()
     }
   }
 
+  private AvaticaParameter createParameter(RelDataTypeField field, RelDataType type)
+  {
+    return new AvaticaParameter(
+        false,
```

Comment:
Would you add a comment on why this is always false?

Reply:
added comment